src/test/java/ruiz/ToDosTest.java

```diff
@@ -0,0 +1,26 @@
+package ruiz;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import ruiz.task.ToDos;
+
+public class ToDosTest {
+    @Test
+    public void saveToString_success() {
+        ToDos test = new ToDos("eat");
+        assertEquals("T | 0 | eat", test.saveTaskString());
```

Comment:
Each unit test should only test one thing. However this test seems to be testing 2 things: 
1. whether the todo is created appropriately
2. whether the todo is able to be marked properly 

Do consider splitting the tests into different test methods.